Nexus.Sdk.Shared/Facades/MailsFacade.cs

```diff
@@ -20,6 +21,18 @@ public async Task<PagedResponse<MailsResponse>> Get(IDictionary<string, string>?
         return await _provider.GetMails(query);
     }
 
+    /// <summary>
+    /// Update Mail status to Sent based on the code
```

Comment:
The comment is for update mail, please change it to create mail :)

Reply:
Done